src/runtime/contrib/arm_compute_lib/acl_runtime.cc

```diff
@@ -166,6 +178,7 @@ class ACLRuntime : public JSONRuntimeBase {
     std::shared_ptr<arm_compute::IFunction> function;
     std::vector<arm_compute::Tensor> inputs;
     std::vector<arm_compute::Tensor> outputs;
+    std::map<std::pair<uint32_t, uint32_t>, uint32_t> json_inputid_to_layer_inputid;
```

Comment:
Perhaps add a comment to explain what this map is for, similar to the commit message :) It would also be worth stating that using it is optional (i.e.only when an operator uses the eid index)